utils/build.mjs

```diff
@@ -65,5 +71,17 @@ export const buildPackage = (name, packageJsons, monorepoName) => {
         yml,
         processActionYml(name, packageJsons, actionYml, monorepoName),
     );
+
+    // Build the JS to the dist folder (so we can support using npm deps)
+    // We need to delete the index file first because `copyDir` would have
+    // copied it to the `dist` folder already but `ncc` won't overwrite an
+    // existing output file.
+    if (fs.existsSync(`${dist}/index.js`)) {
+        fs.rmSync(`${dist}/index.js`, {force: true});
+        execSync(
```

Comment:
It seems like we only want to rm it if it exists - but we should be running the ncc command even if the file doesn't exist, right?

Reply:
Fixed.  We now delete the build file only if it exists, but then only build if a source file exists.